lucien-ios/AddAndEditNewComicBook/ComicFormViewModel.swift

```diff
@@ -65,32 +65,35 @@ class ComicFormViewModel {
     // MARK: - Variables
 
     var comicFormMode: ComicFormMode
-
+    var coverPhoto: UIImage?
     var navigationBarTitle: String {
         return comicFormMode == .add ? "Add Book" : "Edit Book"
     }
+
     var genreTitles: Observable<[String]> {
         return Observable.just(Comic.Genre.allCases.map { $0.title })
     }
+
     var conditionTitles: Observable<[String]> {
         return Observable.just(Comic.Condition.allCases.map { $0.title })
     }
 
-    var coverPhoto: UIImage?
-
     // MARK: - Private Variables
 
     private let disposeBag = DisposeBag()
     private let lucienAPIClient = LucienAPIClient()
+    private let comicID: String?
 
     // ComicFormMode is set to .add by default if default init is used.
     init() {
```

Comment:
Just pointing out that you could get rid of this initializer and just provide default arguments for these two properties.